airflow/cli/commands/sync_perm.py

```diff
@@ -0,0 +1,37 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Sync permission command"""
+import click
+
+from airflow.cli import airflow_cmd
+
+
+@airflow_cmd.command("sync-perm")
+@click.option("--include-dags", is_flag=True, help="If passed, DAG specific permissions will also be synced.")
+def sync_perm(include_dags):
+    """Update permissions for existing roles and optionally DAGs"""
+    from airflow.www.app import cached_app
+
+    appbuilder = cached_app().appbuilder
+    print('Updating actions and resources for all existing roles')
```

Comment:
I also convert `print` invocations to use `rich.console.Console` like this:

```python
from rich.console import Console
...
def sync_perm(...):
    console = Console()
    console.print("Updating actions and resources for all existing roles")
    ...
    if include_dags:
        console.print("Updating permission on all DAG views")
        ...

Reply:
I guess `rich.print` is a shorthand to this : https://rich.readthedocs.io/en/stable/reference/init.html?highlight=rich_print#rich.print.

https://github.com/Textualize/rich/blob/e3fdf3304341e8930ad913be6c67b40ef0e64457/rich/__init__.py#L48-L69

```python
def print(
    *objects: Any,
    sep: str = " ",
    end: str = "\n",
    file: Optional[IO[str]] = None,
    flush: bool = False,
) -> None:
    r"""Print object(s) supplied via positional arguments.
    This function has an identical signature to the built-in print.
    For more advanced features, see the :class:`~rich.console.Console` class.

    Args:
        sep (str, optional): Separator between printed objects. Defaults to " ".
        end (str, optional): Character to write at end of output. Defaults to "\\n".
        file (IO[str], optional): File to write to, or None for stdout. Defaults to None.
        flush (bool, optional): Has no effect as Rich always flushes output. Defaults to False.

    """
    from .console import Console


    write_console = get_console() if file is None else Console(file=file)
    return write_console.print(*objects, sep=sep, end=end)
```